okio/src/appleMain/kotlin/okio/ByteString.kt

```diff
@@ -1,39 +1,221 @@
 /*
- * Copyright (C) 2020 Square, Inc.
+ * Copyright (C) 2018 Square, Inc.
```

Comment:
This old file was deleted, the other file moved here with the same name, and the extension hand written at the bottom. It is not actually changed but git thinks it's being smart.